### PR TITLE
Add DNS wildcard domains KB

### PR DESCRIPTION
**Pull request by Zakaria-Latif**
Add DNS wildcard domains KB

### CODE REVIEW

File: WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/meta.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "Wildcard DNS Abuse Detection": "https://live.paloaltonetworks.com/t5/community-blogs/new-wildcard-dns-abuse-detection-for-dns-security/ba-p/459632"
+  },
+  "privacy_issue": true,
+  "security_issue": true,
+  "categories": {
+    "SOC2_CONTROLS": [
+      "CC_6_7",
+      "CC_6_8",
+      "CC_7_1",
+      "CC_9_1"
```
Comment (PiranhaSa) [inline WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/meta.json:16]:
```suggestion
```

File: WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/meta.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+  "title": "DNS Wildcard Domains",
+  "short_description": "The security risks associated with DNS wildcard domains, including phishing attacks, subdomain takeover, and the bypass of security policies.",
+  "risk_rating": "high",
```
Comment (elyousfi5) [inline WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/meta.json:4]: im not sure if this should be high

Comment (Zakaria-Latif) [reply WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/meta.json:4]: I found that the risk depends on the use case:
- Low Risk: If the wildcard domain is in a controlled environment with strict security policies and proper configuration.
- Medium Risk: If used with weaker controls, exposing potential vulnerabilities.
- High Risk: If misconfigured, leaving subdomains exposed to exploitation, phishing, or data leaks.

Do you think we should lower its risk rating? I tend to go with the highest risk possible.

File: WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/meta.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      "CCPA_1798_120",
+      "CCPA_1798_130",
+      "CCPA_1798_140",
+      "CCPA_1798_150"
+    ],
+    "GDPR": [
+      "ART_32",
+      "ART_24",
+      "ART_5",
+      "ART_12",
```
Comment (ohachimOs) [inline WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/meta.json:35]: I don't see this one

File: WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/meta.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      "CCPA_1798_100",
+      "CCPA_1798_120",
+      "CCPA_1798_130",
+      "CCPA_1798_140",
+      "CCPA_1798_150"
+    ],
+    "GDPR": [
+      "ART_32",
+      "ART_24",
+      "ART_5",
```
Comment (ohachimOs) [inline WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/meta.json:34]: I don't think this one applies

File: WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/description.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+DNS wildcard domains allow a domain owner to specify that all subdomains (or non-existent subdomains) resolve to the same IP address. While this can simplify domain management, it also introduces significant security risks. Attackers can exploit wildcard domains to:
+
+- **Create Phishing Subdomains:** Attackers can easily set up phishing websites using subdomains of a wildcard domain, making them look legitimate.
```
Comment (3asm) [inline WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/description.md:3]: Really, how?

File: WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/description.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+- **Subdomain Takeover:** If the hosting service for a wildcard domain is misconfigured or abandoned, attackers may take control of subdomains and use them for malicious purposes.
```
Comment (3asm) [inline WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/description.md:4]: That's subdomain takeover, a vulnerability on its own, unrelated to wildcards.

**Review by 3asm — CHANGES_REQUESTED** [review]

File: WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/description.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+- **Abuse of Trust:** Users may mistakenly trust all subdomains under a wildcard domain, making them vulnerable to malicious content served from those subdomains.
+
+### Example Scenario:
```
Comment (ErebusZ) [inline WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/description.md:7]: I think the example should contain a vulnerable implementation instead of a scenario

**Pull request closed without merging**